### PR TITLE
Rewrite the guide to be up to date.

### PR DESCRIPTION
**Pull request by AleanaA**
After creating your pull request, tick these boxes if they are applicable to you.

- [ ] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] I have tested my changes on Python 3.5/3.6

----

### Description
The Raspbian guide was hella out of date. This makes it both easier, uses existing repos, and shouldn't need to be changed unless Raspbian gets inevitably fucked up.

### CODE REVIEW

File: _docs/installing/raspbian.md [hunk 1]
```diff
@@ -6,61 +6,21 @@ order: 4
 
 <img class="doc-img" src="{{ site.baseurl }}/images/raspbian.png" alt="Raspbian" style="width: 75px; float: right;"/>
 
-Installing MusicBot on Raspbian for use with a Raspberry Pi 2 or 3B is a **long-winded process and may take a while to complete**. If you're willing to try it, you can run the following commands in order to install it:
+Installing MusicBot on Raspbian for use with a Raspberry Pi 2 or 3B **may take a while to complete and may have poor performance**.
```
Comment (jayktaylor) [inline _docs/installing/raspbian.md]: is "may have poor performance" referring to the install process or the actual bot running or what

Comment (AleanaA) [reply _docs/installing/raspbian.md]: Actual bot running, in reference to the raspi 1, or other lower end raspis.
Wording could probably be changed to remove specifying that it's for rpi 2 and 3b though, as I have no way to test the performance of every different raspi.

**Comment by bmitchell36** [comment]
Command ""c:\users\braiz mitchell\appdata\local\programs\python\python37-32\python.exe" -u -c "import setuptools, tokenize;__file__='C:\\Users\\BRAIZM~1\\AppData\\Local\\Temp\\pip-install-bsx48jb_\\PyNaCl\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\BRAIZM~1\AppData\Local\Temp\pip-record-4gn1cspq\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\BRAIZM~1\AppData\Local\Temp\pip-install-bsx48jb_\PyNaCl\

**Comment by bmitchell36** [comment]
what would i do for that

**Comment by AleanaA** [comment]
Don't use python 3.7, use python 3.5.4 from the  guides for windows. In the
future, please use the support server or make a new issue instead of
commenting on an unrelated pull request.

**Pull request merged**